configuration_manager/xml_parser.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Forschungszentrum Jülich GmbH and Aix-Marseille Université
+# "Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements; and to You under the Apache License,
+#  Version 2.0."
+
+import xml.etree.ElementTree as ET
```

Comment:
I personally do not like two letter acronyms for imports unless they are super common. It could be that ET is the standard way to name it of so... I will need to learn this one.